src/main/java/uk/gov/digital/ho/hocs/casework/domain/repository/CaseTagRepository.java

```diff
@@ -18,6 +18,9 @@ public interface CaseTagRepository extends org.springframework.data.repository.R
     @Modifying
     CaseDataTag save(CaseDataTag entity);
 
+    @Query(value = "select  cdt.*  from  case_data_tag cdt where case_uuid = ?1  and tag = ?2", nativeQuery = true)
```

Comment:
While we do use native queries over the product - I think the general direction is to move to a more JPA way of doing it. You should just need the method to be: 

`CaseDataTag findByCaseUuidAndTag(UUID caseUuid, String tag);`

Without the `Query`...

Reply:
Incorporated and thanks for educating me on JPA side of things.